Firestore plumbing taken care of by FireContainer

yacinehmito opened the pull request.

yacinehmito committed:
Firestore plumbing taken care of by FireContainer

mc100s commented on src/components/FireContainer.js:
```diff
@@ -0,0 +1,78 @@
+  componentWillUnmount() {
+    this.unsync();
+  }
+  saveSnapshot = snapshot => {
```
Experimental feature, change to regular method + bind

yacinehmito committed:
Removed code that used non-standard JS features

saveSnaphsot in FireContainer was a class property set to an arrow
function; is now a regular method bound in the constructor
